### ISSUE TITLE
fails on a certain repo

### ISSUE THREAD

**Issue by karlbrown-va** [CircleCI-Public/CircleCI-Env-Inspector#34]
```
 Fetching Contexts...
  Fetching Projects...
  Fetching Project Variables...
file:///project/node_modules/node-fetch/src/index.js:108
			reject(new FetchError(`request to ${request.url} failed, reason: ${error.message}`, 'system', error));
			       ^

FetchError: request to https://circleci.com/api/v2/project/gh/{redacted}/envvar?page-token= failed, reason: getaddrinfo EAI_AGAIN circleci.com
```

Any idea why this is happening? And can we exclude repos that cause problems?

**Comment by fruit** [comment]
Have the same error. We hundreds of GitHub repositories. Perhaps too many repos?

**Comment by fruit** [comment]
> Same error. We've hundreds of GitHub repositories. Perhaps too many repos?

Issue resolved in latest master (b2d372bd034593d000b743e2974c0974ace7806a)

**Comment by rtau** [comment]
I've experienced this as well and seems this fixed it.

https://github.com/CircleCI-Public/CircleCI-Env-Inspector/pull/38

**Comment by KyleTryon** [comment]
Thank you all for confirming the issue has been resolved. We have taken the fix used in #38 and applied it to the v2 of the app due to release soon to ensure this will not happen in the future. Thank you! Please let us know if you experience any further issues.
